src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -76,6 +76,11 @@ describe('DetailsFieldsComponent', () => {
     description: '',
     technology: '',
   };
+  const dateTest = moment().format('YYYY-MM-DD');
+  const endHourTest = moment().format('HH:mm:ss');
+  const endDateTest = new Date(`${dateTest}T${endHourTest.trim()}`);
+  const startHourTest = moment().subtract(3, 'hours').format('HH:mm:ss');
+  const startDateTest = new Date(`${dateTest}T${startHourTest.trim()}`);
```

Comment:
Do you think we could have any issue about the timing where the tests run? For example, if we put this test running at 1:00 am, due you are subtracting 3 hours, it means that the current hour would be 23:00 of what day?

Reply:
I have deleted that block of lines, it's different now.